lib/Gateway.js

```diff
@@ -753,6 +753,16 @@ Gateway.prototype.discoverDevice = function (node, hassDevice) {
           setId = hassDevice.default_setpoint
         }
 
+        if (payload.max_temp && typeof payload.max_temp === 'string') {
```

Comment:
would be nicer (and more \*cough\* testable \*cough\*) to have this in a function? :)
```js
function myWizzBangFormatter(temp) {
  return temp && typeof temp === 'string' ? this.mqtt.getTopic(this.valueTopic(node, node.values[temp])) : temp
}

payload.min_temp = myWizzBangFormatter(payload.min_temp)
```

\*an `if` statement would probably be more readable than ternary

Reply:
> and more *cough* testable *cough* 

:laughing: 

You right @chrisns, I need to edit my pr anyway as I misundestood what the user was telling me, he wants to take the value and not map the topic (that is not supported in hass)

Reply:
the code still looks basically the same, a function would be way better here

Reply:
when i say the same, both if statements and code are identical apart from variable names that could be parameterized into a function as i'd suggested in my first comment. if you do that i'll even write the test for you before merging ;)